web/app/js/components/ErrorModal.jsx

```diff
@@ -124,12 +124,27 @@ export default class ErrorModal extends React.Component {
     });
   }
 
+  renderIconStatus = errors => {
+    let containers = errors.byPodAndContainer;
+    let showInit = _.flatten(_.map(containers, container => {
+      return _.map(container.byContainer, con => {
+        return con[0].reason;
+      });
+    }));
+    if (_.every(showInit, status => status === "PodInitializing")) {
+      return <Badge status="processing" className="controller-init-icon" onClick={this.showModal} />;
```

Comment:
Since we're waiting for a pod to essentially start, maybe just use a loading icon e.g. `<Icon type="loading" theme="outlined" />`? The processing badge is certainly less alarming than the warning icon, but maybe we don't need to indicate warning at all, if we just want to indicate loading.